packages/auth/test/helpers/integration/helpers.ts

```diff
@@ -17,7 +17,7 @@
 
 import * as sinon from 'sinon';
 import { deleteApp, initializeApp } from '@firebase/app';
-import { Auth, User } from '../../../src/model/public_types';
```

Comment:
Will webpack 5 raise an error without these changes? If so, I'm curious what does the error look like.

Reply:
There's a type mismatch when `getTestInstance()` (line 37 here) is called by tests such as [anonymous.test.ts](https://github.com/firebase/firebase-js-sdk/blob/c9e2b0b8cd5fd0db3cac7bc3a00629ae34302189/packages/auth/test/integration/flows/anonymous.test.ts), which is using the type from `@firebase/auth`. I think it makes sense to do so as this is an integration test using the built package and this is a helper for integration tests..

Reply:
Yeah, make sense. Thanks for the explanation. I see the design doc now.